router/pkg/metric/metric_store.go

```diff
@@ -81,6 +83,11 @@ var (
 		otelmetric.WithUnit("count"),
 		otelmetric.WithDescription(SchemaFieldUsageCounterDescription),
 	}
+
+	RouterConfigDescription = "Current running version of the router configuration"
```

Comment:
Part of this story, we also wanted to get rid of `wg.router.config.version` that reduces cardinality. The user can use the new metric to get global information about the router.

Reply:
As discussed we will be opening a separate ticket.